fixtures/godox/sl300r.json

```diff
@@ -0,0 +1,368 @@
+{
+  "$schema": "https://raw.githubusercontent.com/OpenLightingProject/open-fixture-library/master/schemas/fixture.json",
+  "name": "Godox SL300R",
```

Comment:
The questions about merging the fixtures is beyond me. The definitions always have some fixture-specific information, like the physical and electrical characteristics. Also, sometimes they have unique RMIDs. Maybe those are important to people?

I think it would be great if there was a mechanism to say that all of the channels + modes are the same as another device.

Reply:
It's not often used, but the schema does support putting a `physical` block inside the `modes` block.